source/ui/Image.js

```diff
@@ -173,14 +180,15 @@
 		*/
 		srcChanged: function () {
 			var src = enyo.ri.selectSrc(this.src);
+			var multiple= "url('"+ enyo.path.rewrite(src) +"'), url('"+ this.placeholder + "');";
```

Comment:
Thanks! I was thinking that to properly support image paths set with the `placeholder` property, we would just need to wrap the value in a call to `enyo.path.rewrite`, similar to how we were originally handling the `src` property, to account for $paths and such.